docs/dictionary.txt

```diff
@@ -118,3 +118,9 @@ V1
 VPS
 WIP
 zstd
+Docusaurus
+CloudFlare
+Diátaxis
+OpenRPC
+eg
```

Comment:
_exempli gratia_ should be written `e.g.` https://dictionary.cambridge.org/dictionary/english/eg